homework/index.js

```diff
@@ -0,0 +1,74 @@
+let repositories;
+async function getPromise() {
+  const res = await fetch('https://api.github.com/orgs/HackYourFuture/repos?per_page=100');
+  return res.json();
+}
+
+const myPromise = getPromise(); // Set the promise as a global variable
```

Comment:
You could consider storing the actual data globally, instead of storing the promise globally.

Reply:
Also: my *what kind of* promise? Better name would be `repoPromise` (but as mentioned above; you could refactor so that you no longer store the *promise* globally, but the actual repositories instead).